CONFIG.md

```diff
@@ -1820,14 +1820,49 @@ require'lspconfig'.jsonls.setup{}
 ## julials
 
 https://github.com/julia-vscode/julia-vscode
+
 `LanguageServer.jl` can be installed with `julia` and `Pkg`:
 ```sh
-julia --project=julials -e 'using Pkg; Pkg.add("LanguageServer"); Pkg.add("SymbolServer")'
+julia -e 'using Pkg; Pkg.add("LanguageServer"); Pkg.add("SymbolServer")'
```

Comment:
If you do this, then you can remove the `"--project=" .. server_path`. Since the above line installed `LanguageServer` and `SymbolServer` in the global environment, Julia will be able to run `using LanguageServer` fine without starting Julia with the `--project` flag.

When the custom server_path is used, you'll have to use a `--project` flag. However it should not point to `src`, but instead it should point to a folder with a Project.toml that loads `LanguageServer.jl`. And doesn't have to be the `LanguageServer.jl` repository. The way you've set it up, when you point it to `src`, it loads `LanguageServer.jl` and `SymbolServer.jl` from the global environment, and ignores the `--project=path` flag.

Any folder with a Project.toml (and optionally a Manifest.toml) file is a Julia environment. Previously, the way I set it up, I used a new custom folder for the Julia environment where we install `LanguageServer.jl` and `SymbolServer.jl` so that `LanguageServer.jl` doesn't have to installed in the global environment. Your `server_path` example is great for this, so I'll use that going forward. The `--project` flag must point to a folder with a `Project.toml` though, for this to work as the user intends.

I'm testing it locally, and I'm running into other issues with this set up. Once I figure out what is going on I'll write it up in more detail here. I think the issue I'm having right now is unrelated to this configuration (I'm using Julia nightly and I'm not sure is `LanguageServer.jl` works with that), but I'm trying to figure that out at this point.

Reply:
So, my thoughts on this were to give an explicit example that used whatever the currently active julia environment is, such that if someone . I realize it's global, and did this mostly to align with the other servers that use npm/cargo because it's simpler for people debug. Totally open to changing this back! I think more people would complain if we defaulted to per project with "why isn't my language server installed/launching". It's easy enough to override, but let me know what you think. I'll merge whatever PR you submit with what you believe the most amenable configuration is for JLS users.

Looking into this, I think the issues are on JLS's side regarding the crash (we're not doing anything special on save with newlines as far as I can tell. It's definitely not something with the language client.